core/src/main/java/greencity/controller/SuperAdminController.java

```diff
@@ -144,42 +144,48 @@ public ResponseEntity<GetTariffServiceDto> editTariffService(
      *
      * @param dto  {@link CreateServiceDto}
      * @param uuid {@link String} - user uuid.
-     * @return {@link CreateServiceDto}
-     * @author Vadym Makitra.
+     * @return {@link GetServiceDto}
+     * @author Julia Seti.
```

Comment:
Please do not remove `Vadym Makitra` from `@author`
If you want you could add one more `@author`

Reply:
fixed